docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+  will now have access to the extended behavior for SIMPLE tests, such
+  as being able to exit a test with the WARNING status.
+- Users will now be able to properly run tests based on any Unicode
+  string (as a test reference).
+- Test writers now have access to a test temporary directory that will
+  last not only for the duration of the test, but for the duration of
+  the whole job execution.
+- When running jobs with the TAP plugin enabled (the default), users
+  will now also get a results.tap file created by default in their job
+  results directory.
+- Avocado is now available on Fedora.
+- Avocado packages on PyPI now are always the last version.
+- Avocado test writers can now use a family of decorators, namely
+  `avocado.skip()`, `avocado.skipIf()` and `avocado.skipUnless()` to
+  skip the execution of tests.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins are now distributed in separate packages.
+- The `avocado.Test` class now better exports (and protects) the core
+  class attributes members (such as params and runner_queue).
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- `avocado.utils.linux_modules` functions now returns module name, size,
+  submodules if present, filename, version, number of modules using it,
+  list of modules it is dependent on and finally a list of params.
+- `avocado.TestFail`, `avocado.TestError` and `avocado.TestCancel` are
+  now public Avocado Test APIs, available from the main avocado namespace.
+- Support for new type of test status, `CANCEL`, and of course the
+  mechanisms to set a test with this status.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB plugin.
+- When users run `avocado list --loaders ?` they used to receive a
+  single list containing loader plugins and test types, all mixed
+  together. Now users will get one loader listed per line, along with
+  the test types that each loader supports.
+- Variant-IDs created by the multiplexer are now much more meaningful.
+- The multiplexer now supports the use filters defined inside the YAML
+  files, and greatly expand its filtering capabilities.
+- Avocado now supports resuming jobs that were interrupted.
+- The HTML report now presents the test ID and variant ID in separate
+  columns, allowing users to also sort and filter results based on those
+  specific fields.
+- The HTML report will now show the test parameters used in a test when
+  the user hovers the cursor over the test name.
+- Avocado now reports the total job execution time on the UI, instead of
+  just the tests execution time.
+- The Avocado multiplex command has been renamed to variants.
+- Users will be given more information when a test reference is not
+  recognized by a given test loader.
+- Users can now choose to proceed with the execution of a job even if
```

Comment:
Users can use `--ignore-missing-references` to run a job with undiscovered test references (I think it's important to mention the argument, otherwise people might struggle to find the feature)